internal/pkg/service/common/etcdop/op/atomic_ctx.go

```diff
@@ -0,0 +1,74 @@
+package op
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/keboola/keboola-as-code/internal/pkg/utils/errors"
+)
+
+const (
+	actualAtomicOpCtxKey  = ctxKey("actualAtomicOp")
+	atomicOpMaxReadLevels = 10
+)
+
+type ctxKey string
```

Comment:
Isn't this common type? Do we want to differentiate between context keys? I am not sure what is correct approach.

Reply:
Please read: https://stackoverflow.com/questions/40891345/fix-should-not-use-basic-type-string-as-key-in-context-withvalue-golint